mmdeploy/backend/sdk/export_info.py

```diff
@@ -245,8 +245,9 @@ def get_pipeline(deploy_cfg: mmengine.Config, model_cfg: mmengine.Config,
     task = get_task_type(deploy_cfg)
     input_names = preprocess['input']
     output_names = postprocess['output']
-    if task == Task.CLASSIFICATION or task == Task.SUPER_RESOLUTION \
-            or Task.VIDEO_RECOGNITION:
+    if task in [
```

Comment:
why need to change here? the logic seems unchange.

Reply:
It should be `task == Task.VIDEO_RECOGNITION` not just `Task.VIDEO_RECOGNITION`

Reply:
ooops